actions/subscribers.go

```diff
@@ -371,7 +370,7 @@ func ImportSubscribers(c *gin.Context) {
 	}
 
 	go func(ctx context.Context, client s3iface.S3API, filename string, userID int64, segs []entities.Segment) {
```

Comment:
You need to refactor this action s3 client is created twice once in a middleware and once in the action